vnext/Mso/dispatchQueue/dispatchQueue.h

```diff
@@ -545,10 +538,6 @@ inline /*static*/ DispatchQueue const &DispatchQueue::ConcurrentQueue() noexcept
   return IDispatchQueueStatic::Instance()->ConcurrentQueue();
 }
 
-inline /*static*/ DispatchQueue const &DispatchQueue::MainUIQueue() noexcept {
```

Comment:
>MainUIQueue [](start = 54, length = 11)

Isn't this a break from DevMain? Would DevMain use new pattern in the future?

Reply:
This code is not in the DevMain. It is part of the experimental DispatchQueue 2.0 implementation. At some point I thought that it is a great idea to have such global property, but not any more because we have scenarios in RNW where it does not work. 
So, this experimental feature will never land to DevMain.

---
In reply to: [423973832](https://github.com/microsoft/react-native-windows/pull/4877#discussion_r423973832) [](ancestors = 423973832)